regression-test/data/query_p0/sql_functions/json_function/test_query_json_object.out

```diff
@@ -10,3 +10,70 @@
 {"k0":"k00"}
 {"k1":"k11"}
 
+-- !sql_array --
+{"id":1,"level":["\\"aaa\\"","\\"bbb\\""]}
+
+-- !sql_array --
+{"id":1,"level":["aaa","bbb"]}
+
+-- !sql_array --
+{"id":1,"level":[1,2]}
+
+-- !sql_array --
+{"id":1,"level":[1.1,2.2]}
+
+-- !sql_array --
+{"id":1,"level":[1.1,2.0]}
+
+-- !sql_array --
+{"id":1,"level":[1.0,1.2]}
+
+-- !sql_map --
+{"id":1,"level":{"a":"b","c":"d"}}
+
+-- !sql_map --
+{"id":1,"level":{"a":1,"c":2}}
+
+-- !sql_map --
+{"id":1,"level":{"a":1.1,"c":2.2}}
+
+-- !sql_map --
+{"id":1,"level":{"a":1.1,"c":2.0}}
+
+-- !sql_map --
+{"id":1,"level":{"a":1.0,"c":1.2}}
+
+-- !sql_struct --
+{"id":1,"level":{"name":"a","age":1}}
+
+-- !sql_struct --
+{"id":1,"level":{"name":"a","age":1.1}}
+
+-- !sql_struct --
+{"id":1,"level":{"name":"a","age":1}}
+
+-- !sql_struct --
+{"id":1,"level":{"name":"a","age":1.1}}
+
+-- !sql_json --
+{"id":1,"level":"{\\"a\\":\\"b\\"}"}
+
+-- !sql_json --
+{"id":1,"level":"{\\"a\\":1}"}
+
+-- !sql_json --
+{"id":1,"level":"{\\"a\\":1.1}"}
+
+-- !sql_json --
+{"id":1,"level":"{\\"a\\":1.1}"}
+
+-- !sql_json --
+{"id":1,"level":"{\\"a\\":1.1}"}
```

Comment:
the json output is not a json object, `level ` is string type

Reply:
done!